packages/cli/src/commands/generate-block-types.ts

```diff
@@ -48,12 +49,28 @@ let content = "";
 function writeFieldType(field: BlockMetaField, blockNamePostfix: string) {
     if (field.kind === "String") {
         content += "string";
+
+        if (field.array) {
+            content += "[]";
+        }
     } else if (field.kind === "Number") {
         content += "number";
+
+        if (field.array) {
+            content += "[]";
+        }
     } else if (field.kind === "Boolean") {
         content += "boolean";
+
+        if (field.array) {
+            content += "[]";
+        }
     } else if (field.kind === "Json") {
         content += "unknown";
+
+        if (field.array) {
+            content += "[]";
+        }
```

Comment:
I tried this and `unknown[]` is a valid TS type 😂

Reply:
of course it is :)